inst/doc/Data_Sources_cache/html/__packages

```diff
@@ -0,0 +1,9 @@
+base
+sp
+raster
+zoon
+maxlike
+dismo
+assertthat
+MASS
+spocc
```

Comment:
I also seem to be trying to merge cache chunks in again (which is odd as I didn't commit any of them). If I fix that then hopefully this issue goes away too since its in the same folder